src/mmw/js/src/core/views.js

```diff
@@ -2,7 +2,7 @@
 
 var $ = require('jquery'),
     L = require('leaflet'),
-    leafletLocate = require('leaflet.locatecontrol'),
+    leafletLocate = require('leaflet.locatecontrol'), // jshint ignore:line
```

Comment:
The design pattern we follow for require statements that mutate the environment (but don't return anything useful) is to require without assigning to a variable (Ex: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/core/setup.js#L15).